modules/statics/src/coins.ts

```diff
@@ -3113,6 +3113,14 @@ export const coins = CoinMap.fromCoins([
     '0xf7790914dc335b20aa19d7c9c9171e14e278a134',
     UnderlyingAsset.EURCV
   ),
+  erc20(
+    '4a47b0a6-f5a4-4a74-89f0-2475fb92e191',
+    'eurcvv2',
+    'EUR CoinVertible',
+    18,
+    '0x5F7827FDeb7c20b443265Fc2F40845B715385Ff2',
```

Comment:
the address need to be in lower case

Reply:
Ya, i dont think we use checksum addresses.